client/post-editor/style.scss

```diff
@@ -99,22 +99,22 @@
 	}
 }
 
-.editor__header,
-.editor .mce-container-body,
-.editor .tinymce {
+.post-editor__header,
+.post-editor .mce-container-body,
```

Comment:
I don't know that there'd be any regressions, but to be precise, this should have been updated from `.editor` to `.post-editor__content-editor`. I _think_ we should be okay with the revision though.